lib/src/token.dart

```diff
@@ -223,18 +223,19 @@ class Contact {
       });
 
   factory Contact.fromJson(Map<dynamic, dynamic> json) {
+    print(json);
     return Contact(
       name: json['name'] ?? "",
       emailAddress: json['emailAddress'] ?? "",
       phoneNumber: json['phoneNumber'] ?? "",
       country: json['country'] ?? "",
       postalCode: json['postalCode'] ?? "",
-      state: json['state'] ?? "",
-      city: json['city'] ?? "",
-      street: json['street'] ?? "",
-      isoCountryCode: json['ISOCountryCode'] ?? "",
+      state: json['state'] ?? json['administrativeArea'] ?? "",
+      city: json['city'] ?? json['locality'] ?? "",
+      street: json['street'] ?? json['address1'] != null ? json['address1'] + json['address2'] : "",
```

Comment:
The following line breaks with exception:
```street: json['street'] ?? json['address1'] != null ? json['address1'] + json['address2'] : "",```
`_TypeError (type 'String' is not a subtype of type 'bool')`

Needs parens, so change to 
```json['street'] ?? (json['address1'] != null ? json['address1'] + json['address2'] : "")```